ReactAndroid/src/main/java/com/facebook/react/uimanager/UIViewOperationQueue.java

```diff
@@ -156,16 +156,19 @@ public void execute() {
   private final class ManageChildrenOperation extends ViewOperation {
 
     private final @Nullable int[] mIndicesToRemove;
+    private final @Nullable int[] mTagsToRemove;
     private final @Nullable ViewAtIndex[] mViewsToAdd;
     private final @Nullable int[] mTagsToDelete;
 
     public ManageChildrenOperation(
         int tag,
         @Nullable int[] indicesToRemove,
+        @Nullable int[] tagsToRemove,
         @Nullable ViewAtIndex[] viewsToAdd,
         @Nullable int[] tagsToDelete) {
       super(tag);
       mIndicesToRemove = indicesToRemove;
+      mTagsToRemove = tagsToRemove;
```

Comment:
Why have both `indicesToRemove` and `tagsToRemove`? Isn't `tagsToRemove` sufficient?